src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -141,19 +156,52 @@ public void onTextAvailable(@NotNull final ProcessEvent event, @NotNull final Ke
             getProcessHandler().removeProcessListener(this);
 
             final String urlBase = event.getText().substring(prefix.length());
-            scheduleConnect("ws://" + StringUtil.trimTrailing(urlBase.trim(), '/') + "/ws");
             myOpenObservatoryAction.setUrl("http://" + urlBase);
           }
         }
       });
     }
 
     if (remoteDebug) {
-      LOG.assertTrue(myExecutionResult == null && myDASExecutionContextId == null, myDASExecutionContextId + myExecutionResult);
+      LOG.assertTrue(myExecutionResult == null,  myExecutionResult);
     }
     else {
-      LOG.assertTrue(myExecutionResult != null && myDASExecutionContextId != null, myDASExecutionContextId + myExecutionResult);
+      LOG.assertTrue(myExecutionResult != null, myExecutionResult);
     }
+
+    this.executionEnvironment = executionEnvironment;
+    this.mapper = mapper;
+    myConnector = connector;
+
+    setLogger();
```

Comment:
There is another setLogger() on line 130.

Reply:
good catch. done.